qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -286,12 +295,17 @@ def get_result(item):
                     prob_dict[key] *= 2
                 return prob_dict
             elif isinstance(item, scipy.sparse.spmatrix):
+                # TODO Generalize
```

Comment:
Shall we address the TODO?

Reply:
I believe that this is beyond this PR. Let's discuss offline.

Reply:
Ok